website/blog/2025-02-25-new-ferretdb-v2-rc2-release-with-debian-packages-and-more-indexing-support.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: new-ferretdb-v2-rc2-release-with-debian-packages-and-more-indexing-support
+title: FerretDB releases v2.0.0-rc.2 with new Debian packages and more indexing support
+authors: [alex]
+description: >
+  We have released FerretDB v2.0.0-rc.2, with an embeddable package, new Debian packages, improved stats and logging, and more indexing support.
+image: /img/blog/ferretdb-v2-rc2.jpg
+tags: [release]
+---
+
+![FerretDB v2.0.0-rc.2](/img/blog/ferretdb-v2-rc2.jpg)
+
+[FerretDB](https://www.ferretdb.com/) v2 keeps getting better!
+Based on your feedback, we are making improvements to FerretDB v2 with an embeddable package, new Debian packages, improved logging, and more indexing support.
```

Comment:
```suggestion
Based on your feedback, we are making improvements with an embeddable Go package, new Debian packages, improved logging, and better indexing support.
```

Support can't be "more". And it is unclear what an "embeddable package" is. Is it embeddable into PostgreSQL? No.

The same in the title and description

Reply:
> Fashander marked this conversation as resolved.

https://github.com/FerretDB/FerretDB/blob/d170db2b0798b3035849805f608148b5dfb07124/CONTRIBUTING.md?plain=1#L379

The part about the title and description was ignored.

@Fashander, please focus and review all my comments.